tests/pama-imaging.test.js

```diff
@@ -15,40 +15,48 @@ describe('PAMA Imaging Service Endpoint', () => {
       .send(input)
       .type('json');
 
+    if (response.status !== 200){
+      console.log(response.text)
+    }
+
     expect(response.status).toEqual(200);
-    const { systemActions } = response.body.extension;
-
-    if (systemActions.length > 0) {
```

Comment:
This `if (systemActions.length > 0) {` was causing tests to be skipped anytime no system actions were returned. That led to a bug where sometimes system actions weren't generated, and that automatically caused the tests to be skipped (leading to a pass :/).